Automattic/wp-calypso
# Support content footer: new redesign

## What

We have a new design for the footer of the content on Learn.

## Before

![image](https://user-images.githubusercontent.com/52076348/232747806-63a5091f-e85f-4987-946b-bbe99dc601b2.png)


## After

![image](https://user-images.githubusercontent.com/52076348/232747045-c335fe66-2f62-4eaf-b934-da1e826a1cdb.png)

## Testing
1. Checkout branch
2. `cd apps/happy-blocks` and `yarn dev --sync`
3. Open `learncft.wordpress.com` (remember to sandbox it)
4. Check that the footer of the content respects the design
